### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by RedShadow1983** [Chia-Network/chia-blockchain#5008] [Support]
Hello! I have a problem: I make a raft through the chia blockchain. After upgrading to version 1.1.5 the finished, raft fell off. Chia blockchain writes: the rafts were not found, although there are plots, but work with them is not going on. The raft is created, but it is immediately disabled when it is created. What could be the problem? Nothing I did helped.
![111](https://user-images.githubusercontent.com/84195080/118296294-63e4ad80-b50f-11eb-9794-8d9eff4a2029.png)

**Comment by RedShadow1983** [comment]
![111](https://user-images.githubusercontent.com/84195080/118358095-dc02b000-b5af-11eb-90fc-d46deb60e869.png)

**Comment by lvcivs** [comment]
Try removing and re-adding the plot directory through the GUI. Also check your `config.yaml` to see whether you have any old entries for plot directories.

**Comment by RedShadow1983** [comment]
Hi! I dont know! 15.05.2021, 18:58, "lvcivs" ***@***.***>: Try removing and re-adding the plot directory through the GUI. Also check your config.yaml to see whether you have any old entries for plot directories.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
